src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/stores/DatagridStore.js

```diff
@@ -143,7 +147,7 @@ export default class DatagridStore {
         this.observableOptions.page.set(page);
     }
 
-    @action setActive(active: string | number) {
+    @action setActive(active?: string | number) {
```

Comment:
The question mark belongs to the type, not to the parameter name: `active: ?string | number`